src/ui/map.js

```diff
@@ -268,11 +268,12 @@ const defaultOptions = {
  * @param {boolean} [options.testMode=false] Silences errors and warnings generated due to an invalid accessToken, useful when using the library to write unit tests.
  * @example
  * var map = new mapboxgl.Map({
- *   container: 'map',
- *   center: [-122.420679, 37.772537],
- *   zoom: 13,
- *   style: style_object,
- *   hash: true,
+ *   container: 'map', // container ID
+ *   center: [-122.420679, 37.772537], // starting position [lng, lat]
+ *   zoom: 13, // starting zoom
+ *   style: 'mapbox://styles/mapbox/streets-v11', // style URL or style object
+ *   hash: true, // sync options to URL
```

Comment:
the hash only contains the center, zoom, pitch and bearing. `options` is a much broader set of properties

Reply:
Thank you – it's a helpful nit. Updated the code comment to be more specific. 👍🏼